### DISCUSSION TITLE
Reading js file

### DISCUSSION THREAD

**Discussion by Elmatbaisaad** [symfony/symfony#46693] [Q&A] [answered]
Hello everyone !
i 'm working in a project where i have an ajax request for example when a value == 0 it will display ON but now i want to change this condition the problem is even though i changed it symfony keep the old version of file.
how can i solve that ?
Thank you !

**Comment by DylanKas** [comment, marked as answer]
Are you using webpack encore ?

You need to build asset when updating css/js/images by using:

`yarn encore dev --watch` when using yarn or `npm run watch` when using npm.

This should build your assets (and create a manifest.json referencing those assets).

**Reply by Elmatbaisaad** [reply]
Thank you so much !
